Sources/Nimble/Matchers/Contain.swift

```diff
@@ -9,7 +9,7 @@ private func contain<S: SequenceType, T: Equatable where S.Generator.Element ==
     return NonNilMatcherFunc { actualExpression, failureMessage in
         failureMessage.postfixMessage = "contain <\(arrayAsString(items))>"
         if let actual = try actualExpression.evaluate() {
-            return all(items) {
+            return items.all {
                 return actual.contains($0)
```

Comment:
I've opted for the protocol extension since it lines up with how the stdlib evolved into Swift 2. Also rebased!